cvmfs/sync_union_tarball.h

```diff
@@ -62,12 +65,18 @@ class SyncUnionTarball : public SyncUnion {
   const std::string to_delete_;  ///< entity to delete before to extract the tar
   std::set<std::string>
       know_directories_;  ///< directory that we know already exist
-  std::set<std::string> to_create_catalog_dirs_;
+
   ///< directories where we found catalog marker, after the main traverse we
```

Comment:
This might be broken doxygen syntax.  Multiline comments need to be in the format

    /**
     * Comment
     */